src/components/Blocking/ReportPostAnonymous.js

```diff
@@ -126,7 +127,7 @@ const styles = StyleSheet.create({
     marginBottom: 30,
   },
   btnSkip: {
-    backgroundColor: '#E0E0E0',
+    backgroundColor: COLORS.alto,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Well done on replacing the hardcoded color with a value from the theme. This change promotes code consistency and ease of maintenance.
```diff
- backgroundColor: '#E0E0E0',
+ backgroundColor: COLORS.alto,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->